main.cpp

```diff
@@ -1,15 +1,58 @@
 #include <getopt.h>
 #include <iostream>
-#include <cstdlib>
+#include <string>
+// #include <cstdio>
+// #include <stdexcept>
+// #include <cstdlib>
 
 #include "helpers.h"
 
-using namespace std;
+// using namespace std;
```

Comment:
You could have used `namespace std` instead of writing `std::` everywhere.
What do you think @Himanshu40

Reply:
Yep, since we are not dealing with multiple libs so using std is fine.

Reply:
It is preferred to use ```std::`` over ```using namespace std``` to avoid conflict between user-defined library functions with the same name. That's why I write std:: but in our code, it is fine to use namespace std.

Reply:
yes we are not using any user defined library so better to write `namespace std;`